__tests__/addresses.ts

```diff
@@ -10,118 +10,149 @@ beforeAll(() => {
+const bech32 = "bc1qdx09anw82zhujxzzsn56mruv8qvd33czzy9apt"
+const bech32_caps = "BC1QW508D6QEJXTDG4Y5R3ZARVARY0C5XW7KV8F3T4"
 
-const p2pkh = "1KP2uzAZYoNF6U8BkMBRdivLNujwSjtAQV";
-const p2sh = "3J98t1WpEZ73CNmQviecrnyiWrnqRhWNLy";
-const bech32 = "bc1qdx09anw82zhujxzzsn56mruv8qvd33czzy9apt";
-const bech32_caps = "BC1QW508D6QEJXTDG4Y5R3ZARVARY0C5XW7KV8F3T4";
+const bech32_regtest = "bcrt1qam64av6cyhjsgdwajjpe0l9z8ju4w7rjryecf3"
 
-const bech32_regtest = "bcrt1qam64av6cyhjsgdwajjpe0l9z8ju4w7rjryecf3";
+const bech32_testnet = "tb1q0g444vcyy53pza03zsel3tcwdejt9z5kq3w385aqgazpfxjhsr0qhds7p5"
 
-const bech32_testnet = "tb1q0g444vcyy53pza03zsel3tcwdejt9z5kq3w385aqgazpfxjhsr0qhds7p5";
-
-
-it('bitcoin address mainnet', () => {  
+it("bitcoin address mainnet", () => {
   const network = "mainnet"
-  
+
   checkOnChain(p2pkh, network)
   checkOnChain(p2sh, network)
   checkOnChain(bech32, network)
   checkOnChain(bech32_caps, network)
 
-
   checkOnChainFail(bech32_regtest, network)
-  checkOnChainFail(bech32_testnet, network)  
+  checkOnChainFail(bech32_testnet, network)
 })
 
-it('bitcoin address testnet', () => {  
+it("bitcoin address testnet", () => {
   const network = "testnet"
-  
+
   checkOnChain(bech32_testnet, network)
 
   checkOnChainFail(bech32_regtest, network)
-  checkOnChainFail(bech32, network)  
+  checkOnChainFail(bech32, network)
 })
 
-it('bitcoin address regtest', () => {  
+it("bitcoin address regtest", () => {
   const network = "regtest"
-  
+
   checkOnChain(bech32_regtest, network)
 
   checkOnChainFail(p2pkh, network)
-  checkOnChainFail(bech32_testnet, network)  
+  checkOnChainFail(bech32_testnet, network)
 })
 
-it('amount and amountless', () => {  
+describe("Onchain", () => {
   const network = "mainnet"
-  const address_amount = "bc1qdx09anw82zhujxzzsn56mruv8qvd33czzy9apt?amount=0.00122"
-  const address_no_amount = "bc1qdx09anw82zhujxzzsn56mruv8qvd33czzy9apt"
 
-  {
-    const {valid, paymentType, amountless, amount} = validPayment(address_amount, network, client)
+  it("amount", () => {
+    const address_amount = "bc1qdx09anw82zhujxzzsn56mruv8qvd33czzy9apt?amount=0.00122"
+
+    const { valid, paymentType, amountless, amount } = validPayment(
+      address_amount,
+      network,
+      client,
+    )
     expect(valid).toBeTruthy()
     expect(paymentType).toBe("onchain")
     expect(amount).toBe(122000)
     expect(amountless).toBe(false)
-  }
+  })
+
+  it("amountless", () => {
+    const address_no_amount = "bc1qdx09anw82zhujxzzsn56mruv8qvd33czzy9apt"
 
-  {
-    const {valid, paymentType, amountless, amount} = validPayment(address_no_amount, network, client)
+    const { valid, paymentType, amountless, amount } = validPayment(
+      address_no_amount,
+      network,
+      client,
+    )
     expect(valid).toBeTruthy()
     expect(paymentType).toBe("onchain")
     expect(amount).toBeFalsy()
     expect(amountless).toBe(true)
-  }
+  })
 
+  it("prefix", () => {
```

Comment:
As far as I can tell this is a new test and not just a new formatting.

I'm going to approve to move on but in general can we keep PRs focused to 1 thing. A formatting PR should not add a test.